In [150]:
#lista de 100 numeros
listaCreada = []
for iterable in range(80,201):
    listaCreada.append(iterable)
print(listaCreada)

#uso de la lista previamente creada en el Ejercicio #1
#print(listaCreada)

print(1.0+float(listaCreada.index((listaCreada[-1]))) )
print([i for i, x in enumerate(listaCreada)])
#uso de la lista previamente creada en el Ejercicio #1
#print(listaCreada)

from functools import reduce

promedioLista = (reduce( lambda a,b: a+b/121,listaCreada))

print("El promedio de La lista {} es: \n \nPromedio = {}".format(listaCreada,promedioLista))
print(1.0+float(listaCreada.index((listaCreada[-1]))) )
print([i for i, x in enumerate(listaCreada)])


[80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100, 101, 102, 103, 104, 105, 106, 107, 108, 109, 110, 111, 112, 113, 114, 115, 116, 117, 118, 119, 120, 121, 122, 123, 124, 125, 126, 127, 128, 129, 130, 131, 132, 133, 134, 135, 136, 137, 138, 139, 140, 141, 142, 143, 144, 145, 146, 147, 148, 149, 150, 151, 152, 153, 154, 155, 156, 157, 158, 159, 160, 161, 162, 163, 164, 165, 166, 167, 168, 169, 170, 171, 172, 173, 174, 175, 176, 177, 178, 179, 180, 181, 182, 183, 184, 185, 186, 187, 188, 189, 190, 191, 192, 193, 194, 195, 196, 197, 198, 199, 200]
121.0
[0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100, 101, 102, 10

¡Nota! Esto no es un diagnóstico. Los cálculos que se proporcionan son estimaciones basadas en promedios.
Valor promedio	140
Suma de valor	16940
Cuenta promedio	121
Most Greater Value	200
Most Lowest Value	80
Multply by values	8.815580275870891e+257

In [ ]:
#      for i in notas:
                
                #while i<1 and i>10 and i != 0 :
                 #   if i>=1 and i<=10:
                  #      self.notas = notas.appedn
            
                 #  elif i<1 and i>10: 
                  #      self.notas = notas.appedn(input("Ingrese sus notas, entre 1 y 10, escriba 0 al terminar de ingresarlas: "))
            
                   # else :
                    #    print("Termino la carga de Notas.")
                    
                    
                    
                    
                     
        def get_notas (self):
            
            for i in self.notas:
                while i != 0 :
                    if i>=1 and i<=10:
                        self.notas=notas.appedn(int(input("Ingrese sus notas, entre 1 y 10, escriba 0 al terminar de ingresarlas: ")))
                    elif i<1 and i>10: 
                        self.notas=notas.appedn(int(input("Ingrese sus notas, entre 1 y 10, escriba 0 al terminar de ingresarlas: ")))
                    else :
                        print("Termino la carga de Notas.")
                        
                        
                        
                     
        def get_notas(self):
        
            for i in self.__notas:
                while i != 0 :
                    if i>=1 and i<=10:
                        self.__notas=notas.appedn
                    elif i<1 and i>10: 
                        self.__notas=notas.appedn(int(input("Ingrese sus notas, entre 1 y 10, escriba 0 al terminar de ingresarlas: ")))
                    else :
                        print("Termino la carga de Notas.")
            return self.__notas
            
        def set_notas(self, self__notas):
            return self.__notas   
                        
                        

In [151]:
lista=[input("numeros ")]

print(lista)

numeros 0
['0']
